test/web-worker.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line no-undef
+importScripts('/base/amplitude.js');
+
+var isTrue = function (a) {
```

Comment:
Guessing there isn't a way to import `assert` from mocha here?

Reply:
Correct, the `importScripts` isn't meant to work with modular code which asserts comes with.